playbooks/k8s-services.yml

```diff
@@ -42,6 +42,10 @@
 - hosts: kube-master
   become: true
   tasks:
+    - name: Install required OpenShift packages to support Docker registry
+      include_role:
+        name: openshift
+      run_once: true
```

Comment:
I think we probably need to remove the `run_once` on installing openshift since we can't guarantee which node the next `run_once` will land on. It should be ok to have those packages on every node.

Reply:
Agreed.